api/base/serializers.py

```diff
@@ -143,92 +173,141 @@ def get_attribute(self, obj):
         return field_source_method(auth=auth)
 
 
-class JSONAPIHyperlinkedIdentityField(ser.HyperlinkedIdentityField):
+class RelationshipField(ser.HyperlinkedIdentityField):
```

Comment:
I thought the HyperlinkedIdentityField was doing exactly what we wanted - as it can return a list view URL.  HyperlinkedRelatedFields resolve into a list of detail view URL's. 

https://github.com/CenterForOpenScience/osf.io/pull/3683#discussion_r38497923